packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/sendWithSPAuth.js

```diff
@@ -0,0 +1,62 @@
+/*
+  This sample demonstrates how to create a namespace using TokenCredentials.
```

Comment:
This sample demonstrates how to create a namespace using AAD token credentials gotten from using Service Principal Secrets